app/models/manageiq/providers/redhat/infra_manager/provision_workflow.rb

```diff
@@ -92,4 +92,39 @@ def allowed_storages(options = {})
 
     result.select { |s| s.storage_domain_type == "data" }
   end
+
+  def load_allowed_vlans(hosts, vlans)
+    ems = source_ems
+    if update_vnic_profile_supported?(ems)
```

Comment:
you can probably just call ems.supports_update_vnic_profile? here, unless you plan to have more logic in update_vnic_profile_supported?